### ISSUE TITLE
👨‍🦯 Perform a11y checklist

### ISSUE THREAD

**Issue by boudewijnbout** [boudewijnbout/proof-of-concept#9]
As a developer I want to do the a11y checklist, so that I can see where my page lacks accessebility and/or where it does well. Ultimately I want to visualize where improvements are needed.
